src/controllers/HistoryController.ts

```diff
@@ -131,9 +131,15 @@ export class HistoryController extends RootComponent {
     return diff;
   }
 
-  private getHashValue(value: string): any {
-    Assert.isNonEmptyString(value);
-    return HashUtils.getValue(value, HashUtils.getHash(this.windoh));
+  private getHashValue(key: string): any {
+    Assert.isNonEmptyString(key);
+    let value;
+    try {
+      value = HashUtils.getValue(key, HashUtils.getHash(this.windoh));
+    } catch (error) {
+      this.logger.error(`Could not parse parameter ${key} from URI`);
```

Comment:
When getHashValue returns an undefined value, updateModelFromHash handles it like this: 
```
if (valToSet == undefined) {
  valToSet = this.model.defaultAttributes[key];
}
```

It was already handled since `HashUtils.getValue` can return undefined. I could set the default value directly in `getHashValue` to make it less confusing.

And for the example you provided, everything behaves like you mentioned except that the facet wouldn't exactly be in error. The parameter is simply not parsed so the facet would be in its default state.

I added tests in the Model class to make sure that the parsing does not stop when a value with the wrong type is trying to be set. To complete this PR, I will add a test in HistoryController to verify the behaviour when the parsing fails.